sklearn/manifold/spectral_embedding_.py

```diff
@@ -181,7 +181,7 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
         when using arpack eigen_solver.
 
     norm_laplacian : bool, optional, default=True
-        If True, then compute normalized Laplacian.
+        If True, then compute symmetric normalized Laplacian.
```

Comment:
Not sure that this is pertinent since the parameter does not influence the symmetry.

Reply:
That is true. However, it is unclear otherwise if the algorithm uses the symmetric normalized Laplacian or the random walk normalized Laplacian. Do you have a suggestion for a better spot somewhere to mention this?